const numerador = 100;
const denominador = 0;

try {
    console.log(numerador/denominador)

}catch (error){
    console.log("Erro: "+ error)
}